service/src/main/java/com/ge/predix/acs/attribute/readers/ExternalAttributeReader.java

```diff
@@ -120,7 +120,7 @@ CachedAttributes getAttributesFromAdapters(final String identifier) throws Attri
             try {
                 attributesResponse = this.getAdapterOauth2RestTemplate(adapterConnection).
                         getForEntity(adapterUrl, AttributesResponse.class).getBody();
-            } catch (final Throwable e) {
```

Comment:
Went with approach 1 due to the exception handler being at the controller level which handles that exception